libcst/_nodes/base.py

```diff
@@ -347,6 +365,15 @@ def __repr__(self) -> str:
         lines.append(")")
         return "\n".join(lines)
 
+    @classmethod
+    def field(cls, *args: object, **kwargs: object) -> Any:
+        """
+        A helper that allows us to easily use SimpleWhitespace in dataclass constructor
```

Comment:
This is not only for SimpleWhitespace, other CSTNode also uses it (e.g. TrailingWhitespace).
Can you make the comment more generic?

Reply:
O yeah I missed that when I moved this originally. Good catch!